code/modules/vote/vote_datum.dm

```diff
@@ -106,15 +114,31 @@
 			// If we got here there must only be one thing in the list
 			var/res = winning_options[1]
 
-			if(res in choices)
-				to_chat(world, "<span class='interface'><b><code>[res]</code> won the vote.</b></span>")
-				return res
+			to_chat(world, "<span class='interface'><b><code>[res]</code> won the vote.</b></span>")
+			return res
 
-			to_chat(world, "<span class='interface'>The winner of the vote ([sanitize(res)]) isnt a valid choice? What the heck?</span>")
-			stack_trace("Vote of type [type] concluded with an invalid answer. Answer was [sanitize(res)], choices were [json_encode(choices)]")
-			return null
+		if(VOTE_COUNTING_TICKET)
+			if(!length(voted))
+				to_chat(world, "<span class='interface'>No votes were cast. Do you all hate democracy?!</span>") // shame them
+				return null
+
+			var/lucky_winner_ck = pick(voted)
+			var/lucky_winner_vote = voted[lucky_winner_ck]
+			var/lucky_mob = get_mob_by_ckey(lucky_winner_ck)
+			lucky_winner_ck = safe_get_ckey(lucky_mob)  // anonymize, if requested
```

Comment:
Don't include the mob here, even if anonymised. This will lead to abuse in OOC for "why did you pick that" 

Just have it say reaching into the bag and the answer is xyz

Reply:
(I'm not yet sure how hard it would be to implement, but assuming it's not too hacky) what do you think about anonymizing, yet not completely removing the mob mention?
As in: "...vote cast by a certain medical/security/cargo/service/science department employee" / "a certain ghost" / "a certain Black Terror Spider" / "a certain Blob Overmind" / "a certain Ian" / ...

Reply:
Nah, OOC will still become a mess of "fuck that [mobname]", which will cause issues for simplemobs given:
- Theyre numbered in most cases 
- Mobs like Ian are singletons

Reply:
Numbers can be very easily stripped via `initial(mob.name)` (although honestly, who the hell remembers the mob numbers).

And even for singleton mobs like Ian, the OOC salt would only extend to the current [end of] round, since there's no link between the mob and the player's ckey or "main" IC name. So I see no opportunity for metagrudging.
(And people unhappy with the vote result would OOC salt anyway).

That said, I'm open to total removal of the mob mention. I just want to make sure that it is a somewhat real (and not a purely hypothetical) problem first.